terraform/synapse/synapse-spark-pool/variables.tf

```diff
@@ -36,4 +36,20 @@ variable "module_enabled" {
   type        = bool
   description = "Variable to enable or disable the module."
   default     = true
+}
+
+variable "cache_size" {
+  type        = number
+  description = "(Optional) The cache size in the Spark Pool."
+  default     = 100
+}
+
+variable "spark_version" {
+  type        = number
+  description = "(Optional) The Apache Spark version. Possible values are 2.4 , 3.1 , 3.2 and 3.3. Defaults to 2.4."
+  validation {
+    condition     = contains([2.4, 3.1, 3.2, 3.3], var.spark_version)
+    error_message = "Valid values for sku_name are \"Small\", \"Medium\", \"Large\", \"None\", \"XLarge\", \"XXLarge\", or \"XXXLarge\"."
```

Comment:
The error message in the validation is incorrect. The message should read "Valid values for spark_version are 2.4 , 3.1 , 3.2 and 3.3"